datafusion/sql/tests/cases/plan_to_sql.rs

```diff
@@ -926,12 +926,25 @@ fn test_table_scan_pushdown() -> Result<()> {
     let query_from_table_scan_with_projection = LogicalPlanBuilder::from(
         table_scan(Some("t1"), &schema, Some(vec![0, 1]))?.build()?,
     )
-    .project(vec![wildcard()])?
```

Comment:
Can this actually be a real plan (other than manually constructed) with a wildcard projection and a TableScan that includes only two columns? I would expect them to match.

Reply:
It won't be a real case in DataFusion because the wildcard char will be expanded by `ExpandWildcardRule`

Reply:
@goldmedal - thank you for confirming, good to know!